README.Rmd

```diff
@@ -72,7 +83,8 @@ remotes::install_github("openpharma/mmrm")
 
 ## Getting Started
 
-You can get started by trying out the example:
+See also the [introductory vignette](https://openpharma.github.io/mmrm/main/articles/introduction.html)
```

Comment:
maybe move the mentioning of the vignette further down? I would start here directly with the small example

Reply:
I see the point but I would give readers the option to decide whether they want a "proper" intro or a fast glance. Does not take up much space when u browse on GH but otherwise the link to the "full" intro might not be found - the example is not that short.

Reply:
ok